### ISSUE TITLE
apt update fails for bullseye

### ISSUE THREAD

**Issue by dochench** [debuerreotype/docker-debian-artifacts#116]
Sending build context to Docker daemon   59.9kB
Step 1/7 : FROM debian:bullseye
bullseye: Pulling from library/debian
ffa24fa47fc6: Pull complete
Digest: sha256:484464441fa8f5f8ab84bcefd29a6d654cdc819ddf0f2203cca69b220496f5ad
Status: Downloaded newer image for debian:bullseye
 ---> 804c68b48060
Step 2/7 : RUN apt update
 ---> Running in 21c0b8381206

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Get:1 http://deb.debian.org/debian bullseye InRelease [123 kB]
Get:2 http://security.debian.org/debian-security bullseye-security InRelease [44.1 kB]
Get:3 http://deb.debian.org/debian bullseye-updates InRelease [40.1 kB]
Err:2 http://security.debian.org/debian-security bullseye-security InRelease
  At least one invalid signature was encountered.
Err:1 http://deb.debian.org/debian bullseye InRelease
  At least one invalid signature was encountered.
Err:3 http://deb.debian.org/debian bullseye-updates InRelease
  At least one invalid signature was encountered.
Reading package lists...
W: GPG error: http://security.debian.org/debian-security bullseye-security InRelease: At least one invalid signature was encountered.
E: The repository 'http://security.debian.org/debian-security bullseye-security InRelease' is not signed.
W: GPG error: http://deb.debian.org/debian bullseye InRelease: At least one invalid signature was encountered.
E: The repository 'http://deb.debian.org/debian bullseye InRelease' is not signed.
W: GPG error: http://deb.debian.org/debian bullseye-updates InRelease: At least one invalid signature was encountered.
E: The repository 'http://deb.debian.org/debian bullseye-updates InRelease' is not signed.
The command '/bin/sh -c apt update' returned a non-zero code: 100

**Comment by stappersg** [comment]
@dochench   your mom  is not here.    I did here your cry,  so allow me behave like a good mother and ask:

_What is the idea of this issue?_

**Comment by dochench** [comment]
your build image on docker hub for bullseye is broken for apt update, no? buster updates fine.

**Comment by stappersg** [comment]
On Tue, Feb 09, 2021 at 03:33:43AM -0800, Bruce wrote:
> your build image on docker hub for bullseye is broken for apt update, no? buster updates fine.
> 

Acknowledge

**Comment by tianon** [comment]
Given the image ID you're pulling is armv7, this is a duplicate of #101 / #97 (underlying issue is https://github.com/moby/moby/issues/40734).

TLDR, you need to update libseccomp on your host to 2.4.2 or newer and Docker to 19.03.9 or newer. :+1:

**Comment by dochench** [comment]
Thanks for the pointer on this. Also, learning to search github issue tracker for dups is now on the radar. Sorry for the dup issue.
